api/models/getRecipe.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require("mongoose");
```

Comment:
Can someone explain the significance of this model, additional to the recipe model below?